src/common/item-types/settings-item-type.ts

```diff
@@ -0,0 +1,47 @@
+//////////////////////////////////////////////////////////////////////////////////////////
+//   _  _ ____ _  _ ___  ____                                                           //
+//   |_/  |__| |\ | |  \ |  |    This file belongs to Kando, the cross-platform         //
+//   | \_ |  | | \| |__/ |__|    pie menu. Read more on github.com/menu/kando           //
+//                                                                                      //
+//////////////////////////////////////////////////////////////////////////////////////////
+
+// SPDX-FileCopyrightText: Simon Schneegans <code@simonschneegans.de>
+// SPDX-License-Identifier: MIT
+
+import i18next from 'i18next';
+
+import { IMenuItem } from '../index';
+import { IItemType } from '../item-type-registry';
+
+/** This class provides meta information for the open settings menu item. */
+export class SettingsItemType implements IItemType {
+  get hasChildren(): boolean {
+    return false;
+  }
+
+  get defaultName(): string {
+    return i18next.t('items.settings.name');
+  }
+
+  get defaultIcon(): string {
+    return 'settings';
+  }
+
+  get defaultIconTheme(): string {
+    return 'material-symbols-rounded';
+  }
+
+  get defaultData(): null {
+    return null;
+  }
+
+  get genericDescription(): string {
+    return i18next.t('items.settings.description');
+  }
+
+  getDescription(item: IMenuItem): string {
+    if (item.data == null) {
+      return i18next.t('items.settings.description');
+    }
```

Comment:
Also this if can be removed, I guess:
```suggestion
    return i18next.t('items.settings.description');
```